app/views/refinery/pages/admin/tabs/_copywriting.html.erb

```diff
@@ -2,9 +2,10 @@
 <% if copywriting_phrases and copywriting_phrases.any? %>
   <div class='wym_skin_refinery page_part' id='copywriting-tab'>
     <ul>
-      <% copywriting_phrases.each do |phrase| %>
+      <% copywriting_phrases.group_by(&:scope).each do |phrase| %>
+        <h2><%= phrase.first.humanize %></h2>
         <li>
-          <%= f.fields_for :copywriting_phrases, phrase do |p| %>
+          <%= f.fields_for :copywriting_phrases, phrase.second do |p| %>
```

Comment:
You can skip that by extracting the argument from the block on line 5 by passing an extra argument.

```ruby
# Hash#each convert .to_a when the arity is 1
> {group1: :value, group2: :value}.map {|one| one }
=> [[:group1, :value], [:group2, :value]]
# That mean that you can name the value as `|key, value|` on your block
> {group1: :value, group2: :value}.map {|one,two| one }
=> [:group1, :group2]
```

This mean you can extrapolate the values like this and avoid calling `#second`:

```ruby
copywriting_phrases.group_by(&:scope).each do |_,phrase|
```